wezterm-gui/src/termwindow/charselect.rs

```diff
@@ -575,11 +575,17 @@ impl Modal for CharSelector {
                 self.selection.borrow_mut().clear();
                 self.updated_input();
             }
+            (KeyCode::PageUp, KeyModifiers::NONE) => {
+                self.move_up(6);
```

Comment:
we usually do a fully page at a time for these.  I'd suggest something like:

```suggestion
                self.move_up(*self.max_rows_on_screen.borrow());
```

Reply:
I was thinking about that, but I seem to have a problem with the way the move_down handles the extra logic stuff... it would seem there is not much of a reason/indication that the list jumps/settles my selection at the bottom(but i know why it does in the code). THis is why i didnt want to play with that, and 6 seemed arbitrarily high enough jump.

Reply:
[Peek 2023-04-04 13-36.webm](https://user-images.githubusercontent.com/1828125/229873600-73d63aac-c347-481b-ae63-3ce604f9783d.webm)

Reply:
Thanks for fixing up the jerky movement!

I think this page movement should either be a full page or a half page, rather than an arbitrary fixed number.  My expectation is for full page, but maybe full page minus 1 row would flow better?